scripts/clear-all-fault-leds.sh

```diff
@@ -52,19 +52,31 @@ done
 
 # Now, set the OperationalStatus Functional to what has been requested
 # We filter out the valid object path using a mapper call
+# If we find a unitX/coreX under then, we remove those because they are not mapped to LED's
+# Applicable for CPUs and DIMMs
 if [ ${#excluded_groups} -eq 0 ]
 then
     busctl call xyz.openbmc_project.ObjectMapper /xyz/openbmc_project/object_mapper xyz.openbmc_project.ObjectMapper \
     GetSubTreePaths sias "/xyz/openbmc_project/inventory" 0 1 "xyz.openbmc_project.State.Decorator.OperationalStatus" \
     | sed  's/ /\n/g' | tail -n+3 | awk -F "\"" '{print $2}' | while read -r line
     do
+        echo "$line" | grep "unit\|core" >/dev/null
+        rc=$?
+        if [ $rc -eq 0 ]; then
+            line=$(echo "${line}" | sed -e 's,/[^/]*$,,')
+        fi     
         busctl set-property xyz.openbmc_project.Inventory.Manager "$line" xyz.openbmc_project.State.Decorator.OperationalStatus Functional b "$action";
```

Comment:
Should this logic be moved up into the if logic? i.e. if the line does have unit or core in it, we don't wan to run this busctl do we?

Reply:
No we will not skip . we will just remove unit/core from the dbus object path and that path goes to busctl set-property

Reply:
Hmm, this just isn't computing well for me. Maybe some comments showing examples of what's coming back from the busctl, what the sed/tail/awk is piping to line and what gets passed to busctl? Or maybe a "Tested" section in the commit message? I'm struggling to review this.

Reply:
I have updated the commit message and also the PR comment

Reply:
ok that helps a lot, thanks! Back to my question here. If the line has unit/core in it, then just do a "continue". There is no need to duplicate the busctl set-property X amount of times. Based on your output, we've already set the base path. Skipping the unit/core will save us a large chunk of busctl traffic.

Reply:
Got it! may be i was careless here :-(